scripts/test/async_client.py

```diff
@@ -200,12 +200,14 @@ def kill_children(identifier, params, children):
         if one_child.pid in killed:
             continue
         try:
-            killed.append(one_child.pid)
-            err += add_message_to_report(
-                params,
-                f"{identifier}: killing {one_child.name()} - {str(one_child.pid)}",
-            )
-            one_child.resume()
+            pname = one_child.name()
+            if pname not in ["svchost.exe", "conhost.exe", "mscorsvw.exe"]:
```

Comment:
No... please check instead if the executable starts with "arango" or something like that.

Reply:
No. 
we rather attempt to kill one process to much and have a false positive, than kill one to less, and get no proper results.